locales-da-DK.xml

```diff
@@ -199,8 +199,8 @@
 
     <!-- LONG ROLE FORMS -->
     <term name="director">
-      <single>director</single>
-      <multiple>directors</multiple>
+      <single>instruktør</single>
```

Comment:
I think my answer got lost, so I repeat it:

Directors in film and theatre are "instruktører" in Danish. Directors of musical works are "dirigenter". The ambiguity of the English term creates problems for Danish, but this appears difficukt to resolve.